src/playbacks/no_flash/public/style.scss

```diff
@@ -0,0 +1,25 @@
+[data-no-flash] {
+  z-index: 1000;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  text-align: center;
+}
+
+[data-no-flash] p[data-no-flash-msg] {
+  position: absolute;
+  font-size: 25px;
+  top: 40%;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.